students/scotchwsplenda/Lesson_4/MailRoom_2.py

```diff
@@ -12,40 +12,30 @@
 }
 
 # Opening menu
-prompt = "\n".join(("$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$",
-        "Welcome to the MAILROOM main menu",
-          "Please choose from below options:",
-          "1 - Initiate Groveling",
-          "2 - Data Metrics 3000",
-          "3 - Quit",
-          "4 - Send thank you notes to all donors",
-          "$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$",
-          "Indicate your choice:  "))
+def main():
+    print('''
+              Welcome to the MAILROOM main menu
+              Please choose from below options:
+              1 - Initiate Grovel Sequence
+              2 - Data Metrics 3000
+              3 - Quit
+              4 - Send thank you notes to all donors
+              $$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
+              ''')
+    choice=int(input('Indicate your choice: '))
+    swit_dic={1:send_note, 2:data_metrics, 3:exit_program, 4:mass_mail}
+    swit_dic[choice]()
```

Comment:
I'm not sure why you removed the while True loop?

you should not be manually calling main() function in other functions..